Gulpfile.js

```diff
@@ -40,6 +40,29 @@ var UI_STYLES_FILE_PATH = '/client/ui';
 var UI_STYLES_TEMP_DIR  = path.join('lib', UI_STYLES_FILE_PATH);
 
 
+function promisify (fn) {
```

Comment:
`promisify` from Utils cannot be used in `Gulpfile.js`, because it must be transpiled from ES6 to be used from ES5 code.